compiler/rustc_const_eval/src/interpret/validity.rs

```diff
@@ -34,6 +34,8 @@ use super::{
     Pointer, Projectable, Scalar, ValueVisitor,
 };
 
+// for the validation errors
+#[rustfmt::skip]
```

Comment:
Does it work to put the attribute above the comment? That feels like it may look a bit better, though I am not sure.

Reply:
I checked, it does work.